nbs/models.ipynb

```diff
@@ -61,7 +61,12 @@
     "from statsforecast.arima import auto_arima_f, forecast_arima, fitted_arima\n",
```

Comment:
<p>El link al paper de los modelos Theta nos lleva a otro paper.</p><p>El link correcto es https://www.sciencedirect.com/science/article/pii/S0169207016300243</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='294'/>

Reply:
Thanks! Fixed in [9ffe484](https://github.com/Nixtla/statsforecast/pull/299/commits/9ffe4848d4f8c01c01d630a5b713a59650808bcc)